app/controllers/workflows_controller.rb

```diff
@@ -1,15 +1,53 @@
 # frozen_string_literal: true
 
 class WorkflowsController < Sellers::BaseController
-  before_action :set_body_id_as_app
+  before_action :set_workflow, only: %i[edit emails]
+  before_action :authorize_workflow, only: %i[edit emails]
+
+  layout "inertia"
 
   def index
     authorize Workflow
     create_user_event("workflows_view")
+
+    workflows_presenter = WorkflowsPresenter.new(seller: current_seller)
+    render inertia: "Workflows/Index", props: workflows_presenter.workflows_props
+  end
+
+  def new
+    authorize Workflow
+    create_user_event("workflows_view")
+
+    workflow_presenter = WorkflowPresenter.new(seller: current_seller)
+    render inertia: "Workflows/New", props: workflow_presenter.new_page_react_props
+  end
+
+  def edit
```

Comment:
This is an incomplete migration to Inertia:
- You reused the logic present in `api/internal/workflows#edit`
- That endpoint is no longer used, nor the function `getEditWorkflow` that calls it
- You didn't delete it